### DISCUSSION TITLE
`sort_neighbors` in `dfs_postorder_nodes` and `dfs_preorder_nodes`

### DISCUSSION THREAD

**Discussion by luzianscherrer** [networkx/networkx#7132] [Ideas]
In `bfs_edges` there is an option to define the sort order of the traversal by passing a `sort_neighbors` function. Is there a particular reason why such an option is not existing in `dfs_postorder_nodes` and `dfs_preorder_nodes`? Could this be a possible  new feature or enhancement?

**Comment by dschult** [comment]
I don't believe there is any particular reason this feature is not included in dfs_* except that no one has requested or implemented it. :)

**Comment by Schefflera-Arboricola** [comment]
Hi @luzianscherrer, I created a PR https://github.com/networkx/networkx/pull/7196 for this.

**Reply by luzianscherrer** [reply]
I love it, thank you so much!
